api/mod/src/models/annotation.py

```diff
@@ -50,14 +50,22 @@ class Annotation(AnnotationData):
         description="The full name of the annotator",
         example="Prof. Dr. Folivora",
     )
+    timestamp: int = Field(
+        ...,
+        description="The current timestamp when the annotation is saved",
+        example=420,
```

Comment:
Maybe we should use something more realistic for the example?
(just copy the current timestamp in here :shrug: )